src/sync/pull.rs

```diff
@@ -281,6 +283,21 @@ pub async fn maybe_end_try_pull(
         .await
         .map_err(WriteSyncHeadError)?;
     dag_write.commit().await.map_err(CommitError)?;
+    let (old_last_mutation_id, old_cookie) = Commit::snapshot_meta_parts(&main_snapshot)
```

Comment:
Could these be wrapped in a `debug` check? Seems silly to do this work if we are not going to log them (which we do not do by default)

Reply:
yes, done in https://github.com/rocicorp/repc/pull/327/commits/46d67d234a322306e94c2da01711b7099c56e3a6